cmd/generate/config/rules/nuget.go

```diff
@@ -0,0 +1,29 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func NugetConfigPassword() *config.Rule {
+	r := config.Rule{
+		Description: "Identified a password within a Nuget config file, potentially compromising package management access.",
+		RuleID:      "nuget-config-password",
+		Regex:       regexp.MustCompile(`(?i)(<add key=\"(ClearTextPassword)|(Password)\"\s*)(value=\"([A-Za-z0-9]+)\"\s*)(/>)`),
+		Path:        regexp.MustCompile(`(?i)nuget\.config$`),
+		SecretGroup: 5,
+	}
+
+	tps := map[string]string{
+		"nuget.config": "<add key=\"Password\" value=\"CleartextPassword1\" />",
+		"Nuget.config": "<add key=\"ClearTextPassword\" value=\"CleartextPassword1\" />",
+		"nuget.Config": "<add KEY=\"Password\" value=\"CleartextPassword1\" />",
+	}
+	fps := map[string]string{
+		"some.xml": "<add key=\"Password\" value=\"CleartextPassword1\" />",
+	}
```

Comment:
Additional cases worth adding (random name capitalization required):
```suggestion
	fps := map[string]string{
		"some.xml": "<add key=\"Password\" value=\"CleartextPassword1\" />",
		"nuget.config": `<add key="ClearTextPassword" value="XXXXXXXXXXX" />`, // low entropy
                "Nuget.config": `<add key="ClearTextPassword" value="abc" />`, // too short
                "Nuget.Config": `<add key="ClearTextPassword" value="%TestSourcePassword%" />`, // environment variable
                // known example values
                "NUget.Config": `<add key="ClearTextPassword" value="33f!!lloppa" />`,
                "NUGet.Config": `<add key="ClearTextPassword" value="hal+9ooo_da!sY" />`,
	}
```

Reply:
I extended that, thanks